tutorials/navigation/navigation_introduction_3d.rst

```diff
@@ -0,0 +1,152 @@
+.. _doc_navigation_overview:
```

Comment:
`navigation_introduction_3d` or `navigation_overview`, what shall it be? Same for 2D.

Reminder for the need to check the linking of other pages in the open pr's if this gets changed around, can't remember if this page was crosslinked.